components/paging/pager-load-more.js

```diff
@@ -0,0 +1,155 @@
+import { FocusMixin } from '../../mixins/focus-mixin.js';
+import { FocusVisiblePolyfillMixin } from '../../mixins/focus-visible-polyfill-mixin.js';
+import { getFirstFocusableDescendant } from '../../helpers/focus.js';
+import { labelStyles } from '../typography/styles.js';
+import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
+
+const nativeFocus = document.createElement('div').focus;
+
+/**
+ *  A pager component for load-more paging.
+ * @fires d2l-pager-load-more - Dispatched when the user clicks the load-more button. Consumers must call the provided "complete" method once items have been loaded.
+ */
+class LoadMore extends FocusMixin(FocusVisiblePolyfillMixin(LocalizeCoreElement(LitElement))) {
+
+	static get properties() {
+		return {
+			/**
+			 * Whether there are more items that can be loaded.
+			 * @type {boolean}
+			 */
+			hasMore: { type: Boolean, attribute: 'has-more', reflect: true },
+			/**
+			 * Total number of items. If not specified, neither it nor the count of items showing will be displayed.
+			 * @type {number}
+			 */
+			itemCount: { type: Number, attribute: 'item-count', reflect: true },
+			/**
+			 * The number of additional items to load.
+			 * @type {number}
+			 */
+			pageSize: { type: Number, attribute: 'page-size', reflect: true },
+			/**
+			 * The number of items showing. Assigned by PageableMixin.
+			 * @ignore
+			 */
+			itemShowingCount: { type: Number },
+			_loading: { state: true }
+		};
+	}
+
+	static get styles() {
+		return [ buttonStyles, labelStyles, css`
+			:host {
+				display: block;
+			}
+			:host(:not([has-more])),
+			:host([hidden]) {
+				display: none;
+			}
+			button {
+				align-items: center;
+				background-color: var(--d2l-color-regolith);
+				border: 1px solid var(--d2l-color-sylvite);
+				display: flex;
+				gap: 0.5rem;
+				justify-content: center;
+				width: 100%;
+			}
+			button:hover {
+				background-color: var(--d2l-color-sylvite);
+				border-color: var(--d2l-color-gypsum);
+			}
+			.action {
+				color: var(--d2l-color-celestine);
+			}
+			.separator {
+				border-right: 1px solid var(--d2l-color-mica);
+				height: 0.8rem;
+			}
+			.info {
+				color: var(--d2l-color-galena);
+				font-weight: 400;
+			}
+			d2l-loading-spinner {
+				display: none;
+			}
+			.loading > .action,
+			.loading > .separator,
+			.loading > .info {
+				display: none;
+			}
+			.loading > d2l-loading-spinner {
+				display: inline-block;
+			}
+		`];
+	}
+
+	constructor() {
+		super();
+		this.hasMore = false;
+		this.itemCount = -1;
+		/** @ignore */
+		this.itemShowingCount = 0;
+		this.pageSize = 50;
+		this._loading = false;
+	}
+
+	static get focusElementSelector() {
+		return 'button';
+	}
+
+	render() {
+		if (!this.hasMore) return;
+		const classes = {
+			'd2l-label-text': true,
+			'loading': this._loading
+		};
+		return html`<button class="${classMap(classes)}" @click="${this._handleClick}">
```

Comment:
Otherwise you get a submit button by default!

```suggestion
		return html`<button class="${classMap(classes)}" @click="${this._handleClick}" type="button">
```

Reply:
Thanks... yeah I saw this earlier today in one of your nav PRs. 

@dcostiuc, I think `d2l-tooltip-help` should do the same.

Reply:
Ah, good point, thanks!